src/main/java/com/ita/if103java/ims/service/impl/UpgradeSimpleServiceImpl.java

```diff
@@ -25,6 +30,13 @@ public void upgradeAccount(Long accountId, Long accountTypeId) {
         Integer newLvl = accountTypeDao.findById(accountTypeId).getLevel();
         if (currentLvl < newLvl) {
             accountDao.upgradeAccount(accountId, accountTypeId);
+
+            Event event = new Event();
```

Comment:
@volodymyrvynnyk Could you please create constructor which initializes all below fields, in order to use it different place of the application?